README.md

```diff
@@ -246,6 +246,16 @@ Insert the storage device into the robot's programming pendant and refer to the
 
 ### DX200, YRC1000, and YRC1000micro
 
+If the controller is configured with the Functional Safety Unit (FSU), then `SAVE DATA CRC CHECK FUNC (FSU)` must be temporarily disabled during the installation procedure.
+
+In *Normal Operation* mode:
```

Comment:
Maybe we've already explained this somewhere, but how do you start the controller in "normal operation" mode?